database/src/main/resources/db/migration/V1_60__company_stats.sql

```diff
@@ -0,0 +1,26 @@
+-- Add table for daily stats about company count
+
+CREATE TABLE company_stats (
+  "date" DATE PRIMARY KEY,
+  "count" INTEGER
+);
+
+CREATE OR REPLACE FUNCTION store_daily_company_stats () RETURNS VOID AS $$
+INSERT INTO company_stats ("date", "count")
+VALUES ((SELECT current_date - 1),
+        (SELECT SUM(COALESCE(
```

Comment:
This query doesn't take transport-service.passenger-transportation-info.companies count into account?